orttraining/orttraining/test/python/orttraining_transformer_trainer.py

```diff
@@ -313,7 +394,9 @@ def _prediction_loop(
         eval_losses: List[float] = []
         preds: np.ndarray = None
         label_ids: np.ndarray = None
-        self.model.eval()
+
+        if not self.use_new_api:
```

Comment:
Is this check enough or does it need a check against torch.no_grad() also before setting the model to eval mode? Typically for f/e use cases we want the model to be in training mode.

Reply:
The new frontend does not implement `eval` on ORTTrainer, that is why this is being skipped for the new API